providers/implementations/keymgmt/rsa_kmgmt.c

```diff
@@ -22,12 +22,18 @@
 #include <openssl/params.h>
 #include <openssl/types.h>
 #include "internal/param_build.h"
+#include "prov/providercommon.h"
```

Comment:
Maybe this was a merge error? There is another include 2 lines down..

Reply:
Fixed.  Can I assume that your approval stands?

Reply:
No - see below.. Must of been tired when I reviewed..